ibm/service/logs/README.md

```diff
@@ -0,0 +1,11 @@
+# Terraform IBM Provider 
```

Comment:
We should clearly mention that this is a Beta release and the APIs are expected to change